operator/cmd/mesh/install.go

```diff
@@ -150,35 +154,47 @@ func runApplyCmd(cmd *cobra.Command, rootArgs *rootArgs, iArgs *installArgs, log
 	if err := configLogs(logOpts); err != nil {
 		return fmt.Errorf("could not configure logs: %s", err)
 	}
-	if err := InstallManifests(applyFlagAliases(iArgs.set, iArgs.manifestsPath, iArgs.revision), iArgs.inFilenames, iArgs.force, rootArgs.dryRun,
-		iArgs.kubeConfigPath, iArgs.context, iArgs.readinessTimeout, l); err != nil {
+	iop, err := InstallManifests(applyFlagAliases(iArgs.set, iArgs.manifestsPath, iArgs.revision), iArgs.inFilenames, iArgs.force, rootArgs.dryRun,
+		iArgs.kubeConfigPath, iArgs.context, iArgs.readinessTimeout, l)
+	if err != nil {
 		return fmt.Errorf("failed to install manifests: %v", err)
 	}
 
+	if iArgs.verify {
+		l.LogAndPrint("Verifying installation")
```

Comment:
```suggestion
		l.LogAndPrint("\nVerifying installation:")
```

```
$ istioctl install --verify 
This will install the Istio 1.9.0 profile into the cluster. Proceed? (y/N) y
✔ Istio core installed                                                                                                    
✔ Istiod installed                                                                                                        
✔ Ingress gateways installed                                                                                              
✔ Installation complete                                                                                                   

Verifying installation:
HorizontalPodAutoscaler: istio-ingressgateway.istio-system checked successfully
Deployment: istio-ingressgateway.istio-system checked successfully
```

Reply:
I think it will be good to add a checkmark sign before the verify output. WDYT?
```
$ istioctl install --verify 
This will install the Istio 1.9.0 profile into the cluster. Proceed? (y/N) y
✔ Istio core installed                                                                                                    
✔ Istiod installed                                                                                                        
✔ Ingress gateways installed                                                                                              
✔ Installation complete                                                                                                   

Verifying installation
✔ HorizontalPodAutoscaler: istio-ingressgateway.istio-system checked successfully
✔ Deployment: istio-ingressgateway.istio-system checked successfully
x PodDisruptionBudget: istio-ingressgateway.istio-system failed.
✔ Role: istio-ingressgateway-sds.istio-system checked successfully
✔ RoleBinding: istio-ingressgateway-sds.istio-system checked successfully
...
✔ Istio is installed successfully
```

Reply:
Yes. I agree.

Reply:
Done. When there is an error (in your example `PodDisruptionBudget`) it just stops and prints the error message. So I have not included the cross mark(X).

Reply:
Space is still missing before the `Verifying installation`. It should be following

```
l.LogAndPrint("\nVerifying installation:")
```

```
$ ./out/linux_amd64/istioctl install --revision=1-9-0 --set hub=gcr.io/istio-testing --verify -d manifests/
This will install the Istio 1.9.0 default profile with ["Istio core" "Istiod" "Ingress gateways"] components into the cluster. Proceed? (y/N) y
✔ Istio core installed                                                                                                  
✔ Istiod installed                                                                                                      
✔ Ingress gateways installed                                                                                            
✔ Installation complete                                                                                                 
Verifying installation
✔ ClusterRole: istiod-istio-system.default checked successfully
✔ ClusterRole: istio-reader-istio-system.default checked successfully
✔ ClusterRoleBinding: istio-reader-istio-system.default checked successfully
✔ ClusterRoleBinding: istiod-istio-system.default checked successfully
✔ Role: istiod-istio-system.istio-system checked successfull
```